src/test/java/com/microsoft/sqlserver/jdbc/tvp/TVPResultSetCursorTest.java

```diff
@@ -224,7 +224,18 @@ public void testInvalidStoredProcedureName() throws SQLException {
                 pstmt.execute();
             } catch (SQLException e) {
                 if (!e.getMessage().contains(TestResource.getResource("R_StoredProcedureNotFound"))) {
-                    fail(e.getMessage());
+                    // diff error msg if AE enabled
+                    int start = connectionString.toUpperCase().indexOf(Constants.COLUMN_ENCRYPTION_SETTING);
```

Comment:
It's not a real concern as it's a scenario that'd happen if someone really wanted to break this logic, but our driver takes the last instance of the value. So if you declare columnEncryptionSetting=enabled;columnEncryptionSetting=disabled, this logic would detect it's on even though it's off.

Reply:
Looks like you are trying to check if AE is enabled. You can simply use `ds.getColumnEncryptionSetting()`.